scripts/build/binutils/binutils.sh

```diff
@@ -25,7 +25,7 @@ do_binutils_get() {
             CT_GetCustom "elf2flt" "${CT_ELF2FLT_CUSTOM_VERSION}" \
                 "${CT_ELF2FLT_CUSTOM_LOCATION}"
         else
-            CT_GetGit elf2flt "${CT_ELF2FLT_GIT_CSET}" http://cgit.openadk.org/cgi/cgit/elf2flt.git
+            CT_GetGit elf2flt "${CT_ELF2FLT_GIT_CSET}" https://github.com/uclinux-dev/elf2flt.git
```

Comment:
Well, I asked about cset id because I didn't see it. In binutils.in.2:
````
config ELF2FLT_GIT_CSET
    string
    prompt "git cset"
    default "454b6b415a31959661406bdfbd9dad22229367bf"
````
Cloning the repo and searching:
````
$ git clone https://github.com/uclinux-dev/elf2flt.git
$ cd elf2flt/
$ git log --oneline | grep 454b6b
$
````